07-wt-ed-ds-pipeline.Rmd

```diff
@@ -378,11 +384,11 @@ We have a pretty large data frame! Let's take a quick look.
 dat
 ```
 
-It looks like we have nearly 30,000 observations from 30 variables.
+It looks like we have `r nrow(dat)` observations from 30 variables.
```

Comment:
savvy :)

Reply:
Yeah I wasn’t sure if we should look into why the number ended up being higher than 30,000 when I ran it yesterday. Might be something to look into if someone else does a code review also.